types/src/hard_fork.rs

```diff
@@ -1,6 +1,10 @@
 //! The [`HardFork`] type.
 use std::time::Duration;
 
+use strum::{
+    AsRefStr, Display, EnumCount, EnumIs, EnumString, FromRepr, IntoStaticStr, VariantArray,
+};
```

Comment:
Hopefully pulling in `strum` is justified by replacing the manual "remember to update" code with an automated version.